examples/voice_send.cr

```diff
@@ -50,7 +50,7 @@ client.on_message_create do |payload|
     # The DCAParser class handles parsing of the DCA file. It doesn't do any
     # sending of audio data to Discord itself – that has to be done by
     # VoiceClient.
-    parser = Discord::DCAParser.new(file)
+    parser = Discord::DCAParser.new(file, strict_metadata: false)
```

Comment:
So the example can still play DCA files if their metadata is invalid (since the metadata is irrelevant for the audio playback)